### DISCUSSION TITLE
First time migration does not work

### DISCUSSION THREAD

**Discussion by danielrz** [prisma/prisma#12133] [Q&A] [answered]
hi,

i'm testing Prisma for the first time and it looks very promising.
I have an issue with the first time migration (postgres in docker)
The output of the _npx prisma migrate dev_ command is:

``` 
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "nest", schema "public" at "localhost:5434"

✔ Enter a name for the new migration: … init
Applying migration `20220303124726_init`

The following migration(s) have been created and applied from new schema changes:

migrations/
  └─ 20220303124726_init/
    └─ migration.sql

Your database is now in sync with your schema.

✔ Generated Prisma Client (3.10.0 | library) to ./node_modules/@prisma/client in 114ms
```

But in the Docker log I get this error:

``` 
2022-03-03 12:40:18.901 UTC [1] LOG:  starting PostgreSQL 13.6 (Debian 13.6-1.pgdg110+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2022-03-03 12:40:18.902 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2022-03-03 12:40:18.902 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2022-03-03 12:40:18.903 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2022-03-03 12:40:18.906 UTC [63] LOG:  database system was shut down at 2022-03-03 12:40:18 UTC
2022-03-03 12:40:18.909 UTC [1] LOG:  database system is ready to accept connections
2022-03-03 12:47:19.933 UTC [78] ERROR:  relation "_prisma_migrations" does not exist at character 126
2022-03-03 12:47:19.933 UTC [78] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
```

So for some reasons, it failed to create the _prisma_migrations table.
When I want to execute the prisma studio, the page is not loaded.
Thank you for your help

UPDATED:
I can see the tables in docker:
![image](https://user-images.githubusercontent.com/3119826/156599721-82c2bf77-00b3-4e74-94a5-0bad9087e207.png)
but executing prisma studio does not load the page

**Comment by danielrz** [comment, marked as answer]
OK, I've figured out the Android emulator is using the same prisma studio port: 5555.
It took me a while to figure out because the cli did not mention an error such as port number in use.
After closing the Android emulator, The studio works as expected

**Reply by janpio** [reply]
So the Studio problem seems to be fixed. Could you open an issue for the missing error message over at https://github.com/prisma/studio? Thanks.

Did the error message about the table also resolve itself?

**Reply by danielrz** [reply]
Done regarding Studio: https://github.com/prisma/studio/issues/870
Now regarding the error message about the table, i still see it and it's confusing because actually, the migration has been correctly done and I can see the table _prisma_migrations table in the postgres db

**Reply by janpio** [reply]
That is super weird. Can you also create an issue for that? We might want to reproduce and understand waht is going on there.

**Reply by spigelli** [reply]
Also seeing that issue now

**Reply by f-ramanantsoa** [reply]
having the same error message on my end
and checking the database shows that all tables are there
the seeds didn't apply though
and the command launching my backend didn't run

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
